tests/api/routers/test_filing_api.py

```diff
@@ -122,7 +122,29 @@ async def test_get_latest_submission(self, mocker: MockerFixture, app_fixture: F
         mock.assert_called_with(ANY, "1234567890", "2024")
         assert res.status_code == 204
 
-    async def test_unauthed_patch_filing(self, mocker: MockerFixture, app_fixture: FastAPI):
+    def test_authed_upload_file(
+        self, mocker: MockerFixture, app_fixture: FastAPI, authed_user_mock: Mock, submission_csv: str
+    ):
+        mock_upload = mocker.patch("services.submission_processor.upload_to_storage")
+        mock_upload.return_value = None
+        mock_validate_submission = mocker.patch("services.submission_processor.validate_submission")
+        mock_validate_submission.return_value = None
+        files = {"file": ("submission.csv", open(submission_csv, "rb"))}
+        client = TestClient(app_fixture)
+        res = client.post("/v1/filing/123456790/submissions/1", files=files)
+        assert res.status_code == 202
+
+    def test_unauthed_upload_file(self, mocker: MockerFixture, app_fixture: FastAPI, submission_csv: str):
+        mock_upload = mocker.patch("services.submission_processor.upload_to_storage")
+        mock_upload.return_value = None
+        mock_validate_submission = mocker.patch("services.submission_processor.validate_submission")
+        mock_validate_submission.return_value = None
```

Comment:
Technically these mock return values aren't needed because the unauthed hits and returns from the endpoint prior to any actual processing in the function.